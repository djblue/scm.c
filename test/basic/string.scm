(define str "hello, world")
(write str)
(write (string? str))
(write (string-length str))
(write (string-ref str 0))
(write (string-ref str 1))
(write (string-ref str 2))
(write (string-ref str 3))
(write (string-ref str 4))
(write (string->number "0"))
(write (string->number "1"))
(write (string->number "-1"))
(write (string->list str))
