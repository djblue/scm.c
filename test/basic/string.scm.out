"hello, world"
#t
12
#\h
#\e
#\l
#\l
#\o
0
1
-1
(#\h #\e #\l #\l #\o #\, #\  #\w #\o #\r #\l #\d)
